.travis.yml

```diff
@@ -14,35 +14,26 @@
 # See the License for the specific language governing permissions and
 # limitations under the License.
 
+sudo: required
+
 language: python
+
+# https://github.com/travis-ci/travis-ci/issues/5142#issuecomment-158563862
+
 matrix:
   include:
     - python: "2.7"
       env: REQUIREMENTS=requirements-dev.txt VENV=python2.7
+      services:
+        - docker
     - python: "pypy"
       env: REQUIREMENTS=requirements-dev.txt VENV=pypy
+      services:
+        - docker
     - python: "3.3"
       env: REQUIREMENTS=requirements3-dev.txt VENV=python3.3
-
-before_install: |
-  # needed for the UML container
-  sudo apt-get install -qq -y slirp aufs-tools cgroup-lite
-
-  # prepare docker dir for aufs mount
-  sudo mkdir -p /var/lib/docker
-
-  # download usermode linux kernel for docker
-  curl -sLo linux https://github.com/jpetazzo/sekexe/raw/master/uml
-  chmod +x linux
-
-  # download docker
-  curl -sLo docker \
-    https://get.docker.io/builds/Linux/x86_64/docker-1.3.2
-  chmod +x docker
-
-  # ---------------------------------------------------------------------------
-  chmod +x tests.sh
-  chmod +x run.sh
```

Comment:
I guess the test failures are related to this change?  Was this not working anymore with the new travis setup?